docs/pathTemplate.yml

```diff
@@ -0,0 +1,107 @@
+
+      - `read-buckets`
+      - `write-bucket BUCKET_ID`
+      - `org ORGANIZATION_NAME`
+    
+    `BUCKET_ID` is the ID of the bucket you want to write data to.
+ 
+    <!-- Conditions. Describe any "hidden" or conditional requirements that aren't obvious from requestBody or parameters. -->
+
+    **Limitations**
+
+    <!-- Include the following:
+          - Assumptions about data source, I/O, resource state
+          - Irregularities ("gotchas") in requests, response codes, payloads -->
+    
+    **Rate limits (with InfluxDB Cloud)**
+
+    For InfluxDB Cloud, see [limits and adjustable quotas]({/influxdb/cloud/account-management/limits/).
+   
+    **Related Procedures**
+    
+    <!-- List of links to documented procedures. -->
+
+    - Learn how to [query data with Flux]({{% INFLUXDB_DOCS_URL %}}/influxdb/cloud/query-data/flux/). 
+
+    **Tutorials**
+
+    <!-- List of links to tutorials. -->
+
+    - [Query with the InfluxDB JavaScript client library]({{% INFLUXDB_DOCS_URL %}}/influxdb/cloud/api-guide/client-libraries/nodejs/query/)
+  requestBody:
+    description: | 
+      <!-- Use [CommonMark syntax](https://spec.commonmark.org/0.30/) for descriptions -->
+
+      <!-- Include the following:
+            - What does the body represent? e.g. Task with updates to apply
+            - Do other parameters (headers, query) affect handling of the payload? -->
+
+      Data in line protocol format.
+    content:
+      text/plain:
+        schema:
+          type: string
+          format: byte
+          example:
+            <!-- If the payload can't be constructed from schema properties (e.g. a non-JSON body), provide a sample. -->
+            airSensors,sensor_id=TLM0201 temperature=73.97038159354763,humidity=35.23103248356096,co=0.48445310567793615 1630424257000000000
+            airSensors,sensor_id=TLM0202 temperature=75.30007505999716,humidity=35.651929918691714,co=0.5141876544505826 1630424257000000000       
+  parameters:
+    - in: query
+      name:
+      description: |
+        <!-- Include the following:
+              - Source of the parameter value, how do I find it? e.g., To find the bucket ID, use the [`buckets`]() endpoint
+              - Mutually exclusive with other parameters?
+              - Precedence? E.g. "If you pass values for `bucketId` and `bucket`, InfluxDB uses `bucketId`." --> 
+  responses:
+    '200':
+      content:
+        text/csv:
+          schema:
+            type: string
+            example: >
+              <!-- If the payload can't be constructed from schema properties (e.g. it's a non-JSON body), provide a sample. -->
+              result,table,_start,_stop,_time,region,host,_value
+              mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:00Z,east,A,15.43
+              mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:20Z,east,B,59.25
+              mean,0,2018-05-08T20:50:00Z,2018-05-08T20:51:00Z,2018-05-08T20:50:40Z,east,C,52.62
```

Comment:
is this distinct from lines 84 to 87? diff between text/plain and text/csv?
Looks like this may still be part of _Query with the InfluxDB JavaScript client library_ but I got a little lost

Reply:
> is this distinct from lines 84 to 87? diff between text/plain and text/csv? Looks like this may still be part of _Query with the InfluxDB JavaScript client library_ but I got a little lost

This is an example of a query response (CSV). Line 84 is an example of a write request (line protocol)